torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -230,8 +230,17 @@ struct KinetoThreadLocalState : public ProfilerThreadLocalState {
     }
   }
 
+// Assumption: Total threads number will not exceed 2^16-1, and total ops will not exceed 2^48 -1.
+#define GET_FORWARD_THREAD_KEY(tid, seqNr) \
```

Comment:
We should avoid macros where it's not necessary. Instead something like:

static inline int64_t getForwardThreadKey(...)

Reply:
Thanks. Done.